kmp-library/src/commonTest/kotlin/uk/co/lucasweb/aws/v4/signer/hash/Sha256Test.kt

```diff
@@ -0,0 +1,25 @@
+package uk.co.lucasweb.aws.v4.signer.hash
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class Sha256Test {
```

Comment:
We copied this existing test to the multiplatform library module in order to verify the kotlin SHA-256 implementation behaves the same as the previous one.